docs/_future/getting_started_future.md

```diff
@@ -88,21 +89,32 @@ The application will take a few moments to set up and clone the repository. Once
 🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
 ```
 
-Your app has now been created in the `my-app` directory! This application can be used as a template for you to build your own next-generation Bolt for JavaScript app with. However, if you'd like to test the "Hello World" example included in your template, read on! 📖
+At this point, your app is now ready to go in the `my-app` directory! 
+
+The starter template your app is based off of includes a "Hello World" example that demonstrates how to use the three main building blocks of next-generation Slack apps: [built-in](/bolt-js/future/built-in-functions) and [custom](/bolt-js/future/custom-functions) functions, [triggers](/bolt-js/future/triggers), and [workflows](/bolt-js/future/workflows).
+
+**At a quick glance:**
+1. **Functions** are what define the actions of your app
+2. **Workflows** are made up of **functions**
+3. **Triggers** are how **workflows** are executed
+
+If you'd like to test the "Hello World" example included in your template and see how functions, workflows, and triggers all tie together, read on! 📖
+
+> 💡 Now that it's been set up, this application can be used as starting point for you to build your own next-generation features. You can explore on your own and dive into the code yourself [here](https://github.com/slack-samples/bolt-js-starter-template/tree/future), or continue your journey by diving into [app manifests](/bolt-js/future/app-manifests) or learning how to add more [functions](/bolt-js/future/built-in-functions), [workflows](/bolt-js/future/workflows), and [triggers]() to your app!
 
 ---
 ### Set up your trigger {#setup-trigger}
 
-As mentioned, this app comes with preexisting functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
+Your app comes with a pre-built [workflow](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/workflows/sample-workflow.js) that uses a pre-written [function](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/functions/sample-function.js). This workflow is executed by a trigger defined in [this trigger configuration file](https://github.com/slack-samples/bolt-js-starter-template/blob/future/triggers/hello-world-trigger.json). We can run a command via the CLI to initialize this trigger so that we can run the workflow.
```

Comment:
```suggestion
Your app comes with a sample [workflow](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/workflows/sample-workflow.js) that uses a [function](https://github.com/slack-samples/bolt-js-starter-template/blob/future/manifest/functions/sample-function.js). 

Kick off this workflow using the trigger defined in [this trigger configuration file](https://github.com/slack-samples/bolt-js-starter-template/blob/future/triggers/hello-world-trigger.json). We can run a command via the CLI to initialize this trigger so that we can run the workflow.
```

Some wording nits and maybe we break this into sections?